tx-pool/src/process.rs

```diff
@@ -663,6 +684,20 @@ impl TxPoolService {
             let mut orphan = self.orphan.write().await;
             orphan.remove_orphan_txs(attached.iter().map(|tx| tx.proposal_short_id()));
         }
+
+        // update network fork switch each block
+        {
+            if !self.network.load_ckb2021() {
+                let epoch = self.snapshot().tip_header().epoch().number();
+                if self
+                    .consensus
+                    .hardfork_switch
+                    .is_p2p_network_switch_enabled(epoch)
```

Comment:
Would it be possible that `is_p2p_network_switch_enabled(epoch) returns false` but `is_vm_version_1_and_syscalls_2_enabled(epoch) returns true`?

From [this document](https://hackmd.io/6GCpCIFnSmaYXdgG1lM-zA#%E5%AF%B9%E8%A1%8C%E4%B8%BA%E5%9C%A8-fork-%E5%89%8D%E5%90%8E%E4%BC%9A%E5%8F%91%E7%94%9F%E5%86%B2%E7%AA%81%E7%9A%84%E5%8D%8F%E8%AE%AE), ` is_p2p_network_switch_enabled` and `is_vm_version_1_and_syscalls_2_enabled` should return same value.

Reply:
I think we can ignore this situation, there are two possibilities here:

1. Set epoch error（impossible）
2. Inconsistency caused by the timing of snapshot acquisition （maybe）

Either way, its biggest impact is the loss of some transactions, and the impact on the entire network is negligible

Reply:
I think we could use the same `switch`, since they have to equal to each other.

Reply:
> I think we could use the same `switch`, since they have to equal to each other.

Yes, I agree